internal/notifier/github.go

```diff
@@ -49,10 +55,15 @@ func NewGitHub(addr string, token string) (*GitHub, error) {
 		return nil, fmt.Errorf("invalid repository id %q", id)
 	}
 
-	ctx := context.Background()
 	ts := oauth2.StaticTokenSource(&oauth2.Token{AccessToken: token})
-	tc := oauth2.NewClient(ctx, ts)
+	tc := oauth2.NewClient(context.Background(), ts)
 	client := github.NewClient(tc)
+	if baseUrl.Host != "github.com" {
+		client, err = github.NewEnterpriseClient(host, host, tc)
```

Comment:
Not sure if the api and upload url should be the same. Right now it will be easiest to do it this way, but there may be instances where these are different.